### PR TITLE
Documentation for multiple auto routes feature

### PR DESCRIPTION
**Pull request by dantleech**
Documentation for https://github.com/symfony-cmf/RoutingAuto/pull/62

### CODE REVIEW

File: bundles/routing_auto/introduction.rst [hunk 1]
```diff
@@ -2,8 +2,8 @@
     single: RoutingAuto; Bundles
     single: RoutingAutoBundle
 
-RoutingAutoBundle
-=================
+Introduction
```
Comment (dantleech) [inline bundles/routing_auto/introduction.rst]: Don't know why this was `RoutingAutoBundle`, the `index` already has this title...

**Comment by dantleech** [comment]
The PR for this documentation is still open .. although there is no reason we shouldn't merge it now.

**Comment by dantleech** [comment]
Also this hasn't been reviewed that I am aware of.

**Comment by dbu** [comment]
ups, then please add a [WCM] to the description. the doc looked ok so i thought i just merge it :-/

can you wrap up the feature then?

**Comment by WouterJ** [comment]
it has to be reverted. The feature requires BC breaks and thus will be skipped for 1.1 (in favor of 2.0)

**Comment by dbu** [comment]
reverted in 02a3caf76dc625de11eb9ebdee0b8e492e5750a3 and created #743 to re-add the documentation when ready.

**Pull request merged**